### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.1.1][IntelliJ][Wipro]: Unable to access the sign in dialog .

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1520]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Open your project with IntelliJ IDEA
3. Tab to  and select Tools menu-> "Azure"->"Azure Sign In".
4. In the Azure Sign In window, select "Interactive" radio button, and tab to "Sign In" button and select it.
5. In the Azure Login Dialog window, hit tab and observe keyboard behaviour.

**Expected Result:**
User should be able to tab with keyboard in sign in dialog.

**Actual Result:**
Unable to access the sign in dialog .

**Platform:**
IntelliJ IDEA 2017.3.4 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![sign in dialog not accessible](https://user-images.githubusercontent.com/22956130/36151366-dd262480-107b-11e8-8051-559311ea025c.jpg)

**Comment by Eskibear** [comment]
under further investigation

**Comment by Eskibear** [comment]
similar with #1525 , the issue is probably because the focus cannot be changed to inner web page via `tab`

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749975
